global.go

```diff
@@ -96,7 +93,7 @@ func (gm *globalManager) runAsyncHits() {
 
 			// Send the hits if we reached our batch limit
 			if len(hits) == gm.conf.GlobalBatchLimit {
-				gm.sendHits(ctx, hits)
+				gm.sendHits(context.Background(), hits)
```

Comment:
Why isn't `ctx` wanted?  And if not, might as well remove the function parameter.